target/linux/ar71xx/config-4.9

```diff
@@ -22,18 +22,20 @@ CONFIG_AT803X_PHY=y
 CONFIG_ATH79=y
 # CONFIG_ATH79_DEV_AP9X_PCI is not set
 # CONFIG_ATH79_DEV_DSA is not set
-# CONFIG_ATH79_DEV_ETH is not set
-# CONFIG_ATH79_DEV_GPIO_BUTTONS is not set
-# CONFIG_ATH79_DEV_LEDS_GPIO is not set
-# CONFIG_ATH79_DEV_M25P80 is not set
-# CONFIG_ATH79_DEV_SPI is not set
-# CONFIG_ATH79_DEV_USB is not set
+CONFIG_ATH79_DEV_ETH=y
+CONFIG_ATH79_DEV_GPIO_BUTTONS=y
+CONFIG_ATH79_DEV_LEDS_GPIO=y
+CONFIG_ATH79_DEV_M25P80=y
+CONFIG_ATH79_DEV_SPI=y
+CONFIG_ATH79_DEV_USB=y
+CONFIG_ATH79_DEV_WMAC=y
 # CONFIG_ATH79_MACH_A60 is not set
 # CONFIG_ATH79_MACH_ALFA_AP120C is not set
 # CONFIG_ATH79_MACH_ALFA_AP96 is not set
 # CONFIG_ATH79_MACH_ALFA_NX is not set
 # CONFIG_ATH79_MACH_ALL0258N is not set
 # CONFIG_ATH79_MACH_ALL0315N is not set
+CONFIG_ATH79_MACH_ANONABOX_PRO=y
```

Comment:
Wrong place to enable the support for this device (see target/linux/ar71xx/generic/config-default).